x/bank/v2/keeper/keeper.go

```diff
@@ -22,26 +22,32 @@ import (
 type Keeper struct {
 	appmodulev2.Environment
 
-	authority    []byte
-	addressCodec address.Codec
-	schema       collections.Schema
-	params       collections.Item[types.Params]
-	balances     *collections.IndexedMap[collections.Pair[[]byte, string], math.Int, BalancesIndexes]
-	supply       collections.Map[string, math.Int]
+	accountsKeeper types.AccountsModKeeper
+	authority      []byte
```

Comment:
This is good to have it byte, we want to avoid to have bech32 anywhere in a module. It is basically an sdk.AccAddress (which is a byte alias), but this forces use to the address codec to go to string.